R/t-distribution/app-sam.R

```diff
@@ -262,9 +266,9 @@ server <- function(input, output, session) {
     }
     plot
   })
-  
+
   outputOptions(output, "mode", suspendWhenHidden = FALSE)
   outputOptions(output, "help", suspendWhenHidden = FALSE)
 }
 
-shiny::shinyApp(ui, server)
+shiny::shinyApp(ui, server) %>% shiny::runApp()
```

Comment:
added for non-interactive use (as the file name is non-standard, and doesn't trigger Rstudio black magic integrations